app/main.py

```diff
@@ -1,13 +1,27 @@
-def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+def format_linter_error(err: dict) -> dict:
+    return {("name" if key == "code"
+             else "line" if key == "line_number"
+             else "column" if key == "column_number"
+             else "message" if key == "text"
+             else "source"): ("flake8" if str(value).startswith("./")
+                              else value)
+            for key, value in (dict
+                               (list(err.items())[2:5] + list(err.items())[:2])
+                               ).items()
+            }
```

Comment:
So hard to understand your code. Please simplify it, just return dict with key and values
return {
"line": some value,
"column": some value,